data/json/itemgroups/mall_item_groups.json

```diff
@@ -68,6 +68,8 @@
   {
     "id": "music_shop",
     "type": "item_group",
+    "ammo": 75,
```

Comment:
When you go to a store and buy something that requires batteries, I figured those would be empty on purpose. Batteries not included, and all that.

Reply:
Ah I see, there's didn't seem to be a consistency (e.g. some groups like `consumer_electronics` has it) so I just went across the board.

Sometimes things come with batteries though, perhaps the magazine chance should just be lower?  I wasn't super concerned with balance or anything I just wanted it to be consistent.